src/src/tx_main.cpp

```diff
@@ -488,8 +513,16 @@ void ICACHE_RAM_ATTR timerCallbackNormal()
   // TLM packet reception was the previous slot, transmit this slot (below)
   if (TelemetryRcvPhase == ttrpWindowInProgress)
   {
-    // Stop Receive mode if it is still active
-    Radio.SetTxIdleMode();
+
+#if defined(Regulatory_Domain_EU_CE_2400)
+  if (LBTEnabled) {
+    BeginClearChannelAssessment(); // Stop Receive mode and start LBT
+  } else {
+    Radio.SetTxIdleMode(); // Stop Receive mode if it is still active
+  }
```

Comment:
indentation +2

Reply:
Wow just one thing!

Reply:
```suggestion
  if (LBTEnabled) {
      BeginClearChannelAssessment(); // Stop Receive mode and start LBT
  } else {
      Radio.SetTxIdleMode(); // Stop Receive mode if it is still active
  }
```
or like so:
```suggestion
  if (LBTEnabled)
  {
    BeginClearChannelAssessment(); // Stop Receive mode and start LBT
  } 
  else
  {
    Radio.SetTxIdleMode(); // Stop Receive mode if it is still active
  }
```
I see both 2 and 4 space indentation and both variants of if-else blocks sprinkled around the codebase... I can clean up my contribution to whatever is the preferred standard?

Reply:
@pkendall64 I went with the latter option above. 
@avsaase did you consider moving the LBTEnable check into LBT.cpp for less clutter in tx_main? That could make TX_main look cleaner, but maybe you had a good reason to keep it in tx_main?

Reply:
> @pkendall64 I went with the latter option above.
I think that is the preferred style. We just need to do a code format over the whole codebase at some stage.

Reply:
@avsaase did you consider moving the LBTEnable check into LBT.cpp for less clutter in tx_main? That could make TX_main look cleaner, but maybe you had a good reason to keep it in tx_main?

I kept it together with the domain check without thinking about it too much but now that you mention it I agree it would be better to do the `LBTEnabled` check in LBT.cpp.

Reply:
@avsaase yes, that made it much nicer, check latest commit.

Reply:
> > @pkendall64 I went with the latter option above.
> 
> I think that is the preferred style. We just need to do a code format over the whole codebase at some stage.

agree that this should be the preferred style. Luckily I could move it all into LBT.cpp now.

Reply:
I personally dislike this style with all these lines with only curly brackets. Makes it much harder to see the control flow and find stuff. Inconsistent code style is even worse though so I'm happy to stick to any style in the future. An enforced code style guide would be really nice to have for this project.

Moving the LBT checks to LBT.cpp indeed is a huge improvement. I think this PR is really starting to look good. Time for some reviews.